### PR TITLE
project got a face lift

### PR DESCRIPTION
**Pull request by rbraun04**
Lots of changes to layout on this, didn't touch any javascript and worked mainly with navbar.  Thank you!

### CODE REVIEW

File: TrekAddress.html [hunk 1]
```diff
@@ -18,128 +18,126 @@
...
+        <div class="row">
+            <div class="col-md-2"></div>
 
-</div> 
+            <div class="col-md-8">
+                <div class="firstQuestions">
+                    <form>
+                        <h1 class= "AddressHeader">Starting</h1>
+                        <div class="form-group">
+                            <label for="inputAddress">Address</label>
```
Comment (Koldenblue) [inline TrekAddress.html:51]: I changed the label 'for' fields to match the id fields, there will probably a conflict with the pull request I submitted

**Review by Koldenblue — APPROVED** [review]
see my inline comment about conflict, otherwise likes good

**Comment by Koldenblue** [comment]
Okay actually only one conflict, which I'll let you resolve. Like I mentioned in the inline comments, I really just changed the "for" fields. For example &lt;label for="Inputaddress"&gt;  to &lt;label for="address"&gt;

**Comment by richard2878** [comment]
Nice

**Pull request merged**